README.md

```diff
@@ -1,4 +1,75 @@
 _Note that this project depends on internal Symphony infrastructure (repository.symphony.com), and therefore it can only be built by Symphony LLC employees/partners._
 
-# App-Integrations-Universal
-Universal Webhook Integration
+# Universal Webhook Integration
+The Universal Webhook Integration enables you to send messages directly from any service into a Symphony conversation of your choice - either a 1:1 IM with you, or a room that you are the owner of.
+
+## How it works
+If you have a service that can be configured to send webhooks, all you have to do is point it to the URL you generate in the Universal Webhook Integration configuration pane and then format a message in messageML.
+
+## What formats and events it support and what it produces
```

Comment:
Mention that we also support the XML payload in x-www-form-urlencode format. Content-Type should be application/x-www-form-urlencoded and the messageML payload should be sent in the "payload" form field.